conf/script/ha_setup

```diff
@@ -112,6 +126,25 @@ replace_node() {
 
 # Helping function
 
+config_ha() {
+    cp -rf ${SOURCE_HA_CONF}/ha.conf ${HA_CONF}/ha.conf
+
+    minion1=$(echo $SALT_CLUSTER_SCHEMA | jq '.["local"].node_list[0]' | sed s/\"//g)
+    minion2=$(echo $SALT_CLUSTER_SCHEMA | jq '.["local"].node_list[1]' | sed s/\"//g)
+    uuid_list=$(salt '*' grains.get node_id --output=json)
+    uuid1=$(echo $uuid_list | jq '.["'$minion1'"]' | sed -e s/\"//g -e s/null//g -e '/^\s*$/d')
+    uuid2=$(echo $uuid_list | jq '.["'$minion2'"]' | sed -e s/\"//g -e s/null//g -e '/^\s*$/d')
+    host1=$(echo $SALT_CLUSTER_SCHEMA | jq '.["local"]."'$minion1'".hostname' | sed s/\"//g)
+    host2=$(echo $SALT_CLUSTER_SCHEMA | jq '.["local"]."'$minion2'".hostname' | sed s/\"//g)
+
```

Comment:
It will be good for debug if we add an echo statement printing all the above values.

Reply:
Done